<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxGeneratorWindow</class>
 <widget class="QWidget" name="TxGeneratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generator</string>
  </property>

  <!-- Tree -->
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>Nr</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cycle</string>
    </property>
   </column>
  </widget>

  <!-- Add -->
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>94</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>

  <!-- Remove -->
  <widget class="QPushButton" name="btnRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>94</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>

  <!-- Enable -->
  <widget class="QPushButton" name="btnEnable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>94</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>

  <!-- Disable -->
  <widget class="QPushButton" name="btnDisable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>94</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
